libDF/Cargo.toml

```diff
@@ -116,6 +126,11 @@ tract-pulse = { version = "^0.20.19", optional = true }
 tract-hir = { version = "^0.20.19", optional = true }
 flate2 = { version = "1.0.24", optional = true }
 tar = { version = "0.4.38", optional = true }
+wasm-bindgen = { version = "0.2.87", optional = true}
+getrandom = { version = "0.2", features = ["js"], optional = true}
```

Comment:
Yes. `wasm-bindgen` is needed for wasm compilation, `getrandom` is in truct dependencies (as far as I remembered, but I could be lying). The problem with `getrandom`: you need to build it with `features = ['js']`, otherwise we will get some errors.